src/native/libs/System.Globalization.Native/pal_calendarData.m

```diff
@@ -50,6 +50,35 @@
     return calendarIdentifier;
 }
 
+/*
+Function:
+GetCalendarId
+
+Gets the associated CalendarId for the calendar name.
+*/
+static CalendarId GetCalendarId(const char* calendarName)
+{
+    if (strcasecmp(calendarName, GREGORIAN_NAME) == 0)
+        return GREGORIAN;
+    else if (strcasecmp(calendarName, JAPANESE_NAME) == 0)
+        return JAPAN;
+    else if (strcasecmp(calendarName, BUDDHIST_NAME) == 0)
+        return THAI;
+    else if (strcasecmp(calendarName, HEBREW_NAME) == 0)
+        return HEBREW;
+    else if (strcasecmp(calendarName, DANGI_NAME) == 0)
+        return KOREA;
```

Comment:
This can be removed as there is no native calendar identifier corresponding to the Korean calendar.